core/variables.js

```diff
@@ -232,8 +232,10 @@ Blockly.Variables.generateUniqueName = function(workspace) {
  * @param {function(?string=)=} opt_callback A callback. It will
  *     be passed an acceptable new variable name, or null if change is to be
  *     aborted (cancel button), or undefined if an existing variable was chosen.
+ * @param {?string} opt_type The type of the variable like 'int', 'string', or
+ *     ''. This will default to '', which is a specific none type.
```

Comment:
nit, remove 'none' -> "which is a specific type." is the language we use elsewhere.